metofficedatahub/multiple_files.py

```diff
@@ -236,6 +236,9 @@ def _chunk(dataset: xr.Dataset, *, ideal_chunk_size_mb: float) -> xr.Dataset:
     # Number of floats in a megabyte (assuming 64 bit floats).
     num_float_in_mb = 1024 * 1024 / 8
 
+    # becasue of compressions we can make this number larger
+    num_float_in_mb = num_float_in_mb * 5
```

Comment:
Nipicking but we couldn't done something like
```python
compression_factor = 5
```
And then used that in the `size = ...` line.
This way the variable names are still valid!

Reply:
sure, ill change that